cloud/validations.go

```diff
@@ -104,7 +104,7 @@ func validateCloud(data []byte, jsonSchema *map[string]interface{}) error {
 	formatKeyError := func(invalidKey, similarValidKey string) string {
 		str := fmt.Sprintf("property %s is invalid.", invalidKey)
```

Comment:
^^
This is the one that needed "%q"
Then it would say:
property "of" is invalid. Perhaps you mean "config".

I agree that %q is better than \"%s\" because it handles escaping bad characters as well as quoting.